src/unix/notbsd/linux/musl/b32/arm.rs

```diff
@@ -332,7 +332,5 @@ pub const TIOCCONS: ::c_int = 0x541D;
 pub const SYS_gettid: ::c_long = 224;
 pub const SYS_perf_event_open: ::c_long = 364;
 
-pub const POSIX_MADV_DONTNEED: ::c_int = 0;
-pub const RUSAGE_CHILDREN: ::c_int = 1;
```

Comment:
Perhaps I've misunderstood how the crate is organized? I was under the impression that it would just fall back to the definition in `linux/mod.rs`. Looking now, I see that RUSAGE_CHILDREN isn't there, but with this change the value is the same across all linux platforms (as it should be, since it's part of the kernel ABI), so (unless I've misunderstood how the crate works) could be moved up into mod.rs.

If my understanding is off I can just correct the values in each location.